### PR TITLE
call umount if error happens during mounting

### PR DESCRIPTION
**Pull request by songgao**

### CODE REVIEW

File: mount_darwin.go [hunk 1]
```diff
@@ -198,6 +198,28 @@ func callMount(bin string, daemonVar string, dir string, conf *mountConfig,
...
+	// refuses to laod any page and reboot gets stuck as well.
+	//
+	// We don't have to call unmount if cmd.Start() returns an error because it
+	// only returns an error if starting the process fails.
+	defer func() {
+		if err != nil {
+			log.Printf(
+				"calling syscall.Unmount because callMount failed: %v", err)
+			if err := syscall.Unmount(dir, 0); err != nil {
+				log.Printf("syscall.Unmount failed: %v", err)
```
Comment (strib) [inline mount_darwin.go:218]: Need a `\n` at the end?

**Review by strib — APPROVED** [review]
Thanks!

File: mount_darwin.go (hunk 1 above)
Comment (songgao) [reply mount_darwin.go:218]: Nope! `log` functions appends one automatically if there isn't a trailing `\n` already: https://play.golang.org/p/GJ6ogvz-kRT

**Comment by songgao** [comment]
thanks for the fast response!

File: mount_darwin.go (hunk 1 above)
Comment (strib) [reply mount_darwin.go:218]: D'oh thanks, misread it as `fmt.`.

**Pull request merged**